tests/e2e/service_test.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odoServiceE2e", func() {
+
+	Context("odo component creation", func() {
+		It("should be able to create a service", func() {
+			runCmd("odo service create mysql-persistent")
+			waitForServiceCreateCmd("odo service list | sed 1d", "mysql-persistent", "ProvisionedSuccessfully")
```

Comment:
This check should be probably done using `oc` command not `odo`.  Otherwise, it is exactly the same what is the following test doing.